src/dune_digest/dune_digest_stubs.c

```diff
@@ -1,3 +1,4 @@
+#define CAML_INTERNALS
```

Comment:
Might be worth commenting here that it's needed to access the functions in `md5.h`?

Reply:
Fair enough. I added a comment in 818a6c3